app/javascript/app/FeedbackForm.js

```diff
@@ -41,13 +41,38 @@ function FeedbackForm() {
     return this;
   }
 
+  function feedbackFormSend() {
+    const agent = navigator.userAgent;
+
+    const csrfToken = document.querySelector('meta[name="csrf-token"]').getAttribute('content');
+
+    const transmission = {
+      message: _commentInput.value,
+      from: _emailInput.value,
+      agent,
+    };
+
+    const xhr = new XMLHttpRequest();
+    xhr.open('POST', '/feedback');
+    xhr.setRequestHeader('Content-Type', 'application/json');
+    xhr.setRequestHeader('X-CSRF-Token', csrfToken);
+    xhr.onload = function() {
+      if (xhr.status === 200) {
+        _onSuccess();
+      } else if (xhr.status !== 200) {
```

Comment:
Would we want to error on anything other than 200 status? Maybe just an `else` statement here?

Reply:
Good point. I will fix.